src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -1,23 +1,34 @@
 package frc.robot.subsystems;
 
+import com.chopshop166.chopshoplib.outputs.IDSolenoid;
 import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
 
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.StartEndCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
 
+//The intake takes in the balls from the ground, stores them and relays that number, then it puts them into the shooter(if ball goes into storage area, then it adds that number to a storage system)
```

Comment:
I think we want the intake subsystem to only cover the mechanism that flips down out of the robot and intakes the power cells.

The rest of this behavior should be included in the `indexer` subsystem.